packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -473,6 +475,11 @@ class CheckboxListTile extends StatelessWidget {
   // the default value to true.
   final bool internalAddSemanticForOnTap;
 
+  /// Defines the scale for Transform.scale
+  ///
+  /// When the value is null, the [checkboxScaleFactor] is `1.0`.
```

Comment:
Nit: replace with: "If a value is not provided, a default of `1.0` will be used.

"When the value is null" suggests that it property itself is nullable.

Reply:
fixed